proposal/more-math-functions.md

```diff
@@ -175,8 +176,12 @@ tan($number)
 
 * If `$number` has units but is not an angle, throw an error.
 * If `$number` is unitless, treat it as though its unit were `rad`.
-* If `$number == Infinity`, return `NaN` as a unitless number.
+* If `$number == Infinity` or `$number == -Infinity`, return `NaN` as a unitless number.
 * If `$number == -0`, return `-0` as a unitless number.
+* If `$number` has units `rad` and is equal to `0.5rad * pi +/- 2rad * pi * n`,
+  where `n` is any integer, return `16331239353195370` as a unitless number.
```

Comment:
Why these huge constants and not `Infinity`/`-Infinity`?

Reply:
The CSS spec basically says that `rad` can't exactly represent the asymptotic values, so "whether the result is a very large negative or positive value can depend on rounding and precise details of how numbers are internally stored. It’s recommended you don’t depend on this behavior if using such units."

We could take a stance here and just have these yield `Infinity`/`-Infinity`?

Reply:
I'd prefer that, I think, yeah.